assets/app/views/directives/_truncate.html

```diff
@@ -0,0 +1 @@
+<span ng-attr-title="{{content}}">{{content | limitTo : limit}}<span ng-if="content.length > (limit | toInt)">&hellip;</span></span>
```

Comment:
Yeah, doesn't work without coercion doesn't work since `limit` is a string when it's passed as an attribute to the directive. There might be a cleaner way.

I'll look at not chopping words.